sw/device/silicon_creator/manuf/data/manuf_testplan.hjson

```diff
@@ -0,0 +1,220 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+{
+  name: "manufacturing"
+
+  testpoints: [
+    {
+      name: manuf_cp_unlock_raw
+      desc: '''Verify transition from RAW to TEST_UNLOCKED lc_state.
+
+            - Pre-load OTP with RAW lc_state.
+            - Bring device out of reset selecting the Life Cycle (LC) TAP interface using strap pins.
+
+            Perfom the following steps via LC TAP interface:
+
+            - Provide RAW_UNLOCK.
+            - Poll for lc_ctrl STATUS register for TRANSITION_SUCCESSFUL. Check for errors or timeout.
+            - Perform reset by toggling the POR pin.
+            - Verify the lc_state is TEST_UNLOCKED0.
+
+            Repeat the test case with TRANSITION_CTRL.EXT_CLOCK_EN set.
+
+            The manufacturing environment is required to support test reruns whenever possible:
+
+            - Verify that the RAW_UNLOCK process can be re-tried after failed or interrupted attempts.
+            '''
+      tags: ["fpga", "silicon"]
+      stage: V2
+      tests: []
+    }
+
+    {
+      name: manuf_scrap
+      desc: '''Verify transition from any state to SCRAP lc_state.
+
+            Manufacturing tests should be able to move the device from any lc_state into SCRAP state.
+
+            For each manufacturing lc_state `s`:
+            - Pre-load OTP with `s` lc_state image.
+            - Bring device out of reset selecting the Life Cycle (LC) TAP interface using strap pins.
+            - Request transition to SCRAP state via LC TAP interface.
+            - Perform reset by togglig the POR pin.
+            - Check we are in SCRAP state via LC TAP interface.
+
+            The manufacturing environment is required to support test reruns whenever possible:
+
+            - Verify that the transition to SCRAP mode can be re-tried after failed or interrupted attempts.
```

Comment:
Note that depending on when exactly the process is interrupted, the LC state vector inside OTP is corrupt and will hence decode to INVALID. I guess this does not matter much for SCRAP (because the effect is the same), but it matters for other LC state transition targets.